src/assets/translations/en-us.yaml

```diff
@@ -319,7 +319,7 @@ authConfig:
       app:
         label: Application name
         value: 'Anything you like, e.g. My {vendor}'
-      calllback:
+      callback:
```

Comment:
Not sure about things like this.

Sometimes people have intentional typos. I've left some alone in this repository.

Reply:
Hi! Please skip checking the translations from `/src/assets/translations/…`; those are imported from [dashboard](https://github.com/rancher/dashboard/blob/master/assets/translations/en-us.yaml) and fixes should be made there (we're only using that at the moment for components we imported).